src/query/sql/src/planner/utils.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Datafuse Labs.
+//
```

Comment:
Avoid using the file name `utils.rs`,  it will be fat and fat. Please choose a meaningful name for the file.